sdk/cosmos/azure-cosmos-table/pom.xml

```diff
@@ -0,0 +1,107 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-cosmos-table</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos;current} -->
+  <name>Microsoft Azure SDK for Azure Table</name>
+  <description>This Package contains Microsoft Azure Cosmos Table</description>
+  <packaging>jar</packaging>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+    <connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
+    <tag>HEAD</tag>
+  </scm>
+
+  <!-- CosmosSkip - Needed temporary values to 10% not fail. -->
```

Comment:
Suggestion: You should probably change this CosmosSkip to CosmosTableSkip, the CosmosSkip is something specific for the Cosmos team and I wouldn't want the one in here to get mixed up with the other item.